src/commands/BootstrapCommand.js

```diff
@@ -100,150 +121,147 @@ export default class BootstrapCommand extends Command {
-      : dep.bin;
-
-    async.series([(cb) => FileSystemUtilities.mkdirp(dest, cb)].concat(
-      Object.keys(bins).map((name) => (cb) => FileSystemUtilities.symlink(
-        path.join(dep.location, bins[name]),
-        path.join(dest, name),
-        cb
-      ))
-    ), callback);
-  }
-
-  installExternalPackages(pkg, callback) {
-    const allDependencies = pkg.allDependencies;
-
-    const externalPackages = Object.keys(allDependencies)
-      .filter((dependency) => {
-        const match = find(this.packages, (pkg) => {
-          return pkg.name === dependency;
+  /**
+   * Symlink all packages to the packages/node_modules directory
+   * Symlink package binaries to dependent packages' node_modules/.bin directory
+   * @param {Function} callback
+   */
+  symlinkPackages(callback) {
+    this.logger.info("Symlinking packages and binaries");
+    this.progressBar.init(this.filteredPackages.length);
+    const actions = [];
+    this.filteredPackages.forEach((filteredPackage) => {
+      // actions to run for this package
+      const packageActions = [];
+      Object.keys(filteredPackage.allDependencies)
+        // filter out external dependencies and incompatible packages
+        .filter((dependency) => {
+          const match = this.packageGraph.get(dependency);
+          return match && filteredPackage.hasMatchingDependency(match.package);
+        })
+        .forEach((dependency) => {
+          // get Package of dependency
+          const dependencyPackage = this.packageGraph.get(dependency).package;
+          // get path to dependency and its scope
+          const { location: dependencyLocation } = dependencyPackage;
+          const dependencyPackageJsonLocation = path.join(dependencyLocation, "package.json");
+          // ignore dependencies without a package.json file
+          if (!FileSystemUtilities.existsSync(dependencyPackageJsonLocation)) {
+            this.logger.error(
+              `Unable to find package.json for ${dependency} dependency of ${filteredPackage.name},  ` +
+              "Skipping..."
+            );
+          } else {
+            // get the destination directory name of the dependency
+            const pkgDependencyLocation = path.join(filteredPackage.nodeModulesLocation, dependencyPackage.name);
+            // check if dependency is already installed
+            if (FileSystemUtilities.existsSync(pkgDependencyLocation)) {
+              const isDepSymlink = FileSystemUtilities.isSymlink(pkgDependencyLocation);
+              // installed dependency is a symlink pointing to a different location
+              if (isDepSymlink !== false && isDepSymlink !== dependencyLocation) {
+                this.logger.warning(
+                  `Symlink already exists for ${dependency} dependency of ${filteredPackage.name}, ` +
+                  "but links to different location. Replacing with updated symlink..."
+                );
+              // installed dependency is not a symlink
```

Comment:
what's weird is that `akutil-shared-styles` is installed normally during the `Installing external dependencies...` step.